css/giving-rides.css

```diff
@@ -0,0 +1,304 @@
+    margin-right: 27rem;
+    margin-left: 27rem;
+    padding: 0;
+    font-size: 1.6rem;
+    display:flex;
+    flex-direction: row;
+    justify-content: space-around;
+}
+.info-icon-box{
+    text-align: center;
+    color: #807C7C;
+    background-color: #F1F1F1;
+    width: 30%;
+    /* height: 40rem; */
+}
+.info-icon-box-gray{
+    text-align: center;
+    color: #807C7C;
+    background-color: #F1F1F1;
+    width:50%;
+    /* height: 40rem; */
+}
+.info-icon-box-white{
+    text-align: center;
+    color: #807C7C;
+    resize: none;
+    background-color: #FFFFFF;
+    width:50%;
+    /* height: 40rem; */
+}
+.info-icon-text{
+    margin-left: 1.2rem;
+    margin-right: 1.2rem;
+    margin-bottom: 2rem;
+}
+.info-icon-text-subtitle{
+    margin-left: 0rem;
+    margin-right: 0rem;
+    margin-bottom: 2rem;
+    color: #000000;
+}
+.info-icon-image{
+    margin-left:auto;
+    margin-right:auto;
+    margin-top: -5rem;
+    margin-bottom: 2rem;
+    width:11rem;
+}
+.box-bot{
+    display: flex;
+    width:50%;
+    flex-direction: row;
+}
+.box-top{
+    display: flex;
+    width:50%;
+    flex-direction: row;
+
+}
+
+@media only screen and (max-width: 1300px) {
+    #info-list{
+        /* font-size: 1.4rem; */
+        margin-left: 20rem;
+        margin-right: 20rem;
+    }
+    .info-icon-image{
+        margin-top: -4rem;
+        margin-bottom: 2rem;
+        width:9rem;
+    }
+    .info-icon-list{
+        margin-top: 8rem;
+        /* font-size: 1.4rem; */
+        margin-right: 17rem;
+        margin-left: 17rem;
+    }
+    .info-icon-list-home{
+        margin-right: 13rem;
+        margin-left: 13rem;
+    }
+    .important-info{
+        padding-left:8rem; 
+    }
+}
+@media only screen and (max-width: 950px) {
+    #info-list{
+        /* font-size: 1.4rem; */
+        margin-left: 13rem;
+        margin-right: 13rem;
+    }
+    .info-icon-image{
+        margin-top: -3rem;
+        margin-bottom: 2rem;
+        width:7rem;
+    }
+    .info-icon-list{
+        margin-top: 8rem;
+        /* font-size: 1.4rem; */
+        margin-right: 10rem;
+        margin-left: 10rem;
+    }
+    .info-icon-text{
+        /* font-size: 1.2rem; */
+    }
+    .info-icon-list-home{
+        margin-right: 9rem;
+        margin-left: 9rem;
+    }
+    .important-info{
+        padding-left:7rem; 
+    }
+}
+
+@media only screen and (max-width: 825px) {
+    #info-list{
+        /* font-size: 1.3rem; */
+        margin-left: 10rem;
+        margin-right: 10rem;
+    }
+    .info-icon-image{
+        margin-top: -2rem;
+        margin-bottom: 2rem;
+        width:5rem;
+    }
+    .info-icon-list{
+        margin-top: 7rem;
+        /* font-size: 1.3rem; */
+        margin-right: 7rem;
+        margin-left: 7rem;
+    }
+    .info-icon-text{
+        /* font-size: 1.1rem; */
+    }
+    .info-icon-list-home{
+        margin-right: 6rem;
+        margin-left: 6rem;
+    }
+    .important-info{
+        padding-left:5rem; 
+    }
+}
+
```

Comment:
You shouldn't need most of the code in this media queries. Think about how you can just center the list and the boxes so that you don't need to mess with the padding and margins at each step. Flex can also help handle the spacing between the boxes as it gets more responsive. Check out my comments on the lines above and where you can change some code so you won't need these queries